x-pack/test/api_integration/apis/monitoring/elasticsearch/nodes_mb.js

```diff
@@ -7,341 +7,43 @@
 
 import expect from '@kbn/expect';
 import nodesListingFixtureGreen from './fixtures/nodes_listing_green.json';
-import nodesListingFixtureRed from './fixtures/nodes_listing_red.json';
-import nodesListingFixtureCgroup from './fixtures/nodes_listing_cgroup.json';
-import nodesListingFixturePagination from './fixtures/nodes_listing_pagination.json';
 import { getLifecycleMethods } from '../data_stream';
 
 export default function ({ getService }) {
   const supertest = getService('supertest');
-  const esArchiver = getService('esArchiver');
   const { setup, tearDown } = getLifecycleMethods(getService);
 
-  describe('nodes mb', () => {
-    describe('with green platinum cluster', () => {
-      const archive =
-        'x-pack/test/functional/es_archives/monitoring/singlecluster_green_platinum_mb';
-      const timeRange = {
-        min: '2018-02-13T17:04:50.000Z',
-        max: '2018-02-13T17:51:55.000Z',
-      };
-      const pagination = {
-        size: 10,
-        index: 0,
-      };
-
-      before('load clusters archive', () => {
-        return setup(archive);
-      });
-
-      after('unload clusters archive', () => {
-        return tearDown();
-      });
-
-      it('should return data for 2 active nodes', async () => {
-        const { body } = await supertest
-          .post('/api/monitoring/v1/clusters/fHJwISmKTFO8bj57oFBLUQ/elasticsearch/nodes')
-          .set('kbn-xsrf', 'xxx')
-          .send({ timeRange, pagination })
-          .expect(200);
-        expect(body).to.eql(nodesListingFixtureGreen);
-      });
-    });
-
-    describe('with red platinum cluster', () => {
```

Comment:
These tests are duplicates of the legacy ones that are not testing metricbeat data. I've attempted to use the corresponding _mb archive to no avail so we have to rewrite if we want to keep them.
I've opted to remove them without replacement because they're only testing variants of the same API, and the first test guarantees that metricbeat/package data is properly loaded